ait/commons/util/command/sync.py

```diff
@@ -60,22 +60,20 @@ def transfer(f):
                 try:
                     
                     fname = f.key[37:]
-                    contentType = ''
-                    obj_ = s3_cli.head_object(Bucket=self.aws.bucket_name, Key=f.key)
-                    if obj_ and obj_['ContentType']:
-                        contentType = obj_['ContentType']
-                        if "dcp-type=data" not in contentType:
-                            contentType += '; dcp-type=data'
```

Comment:
it might be better to retain the actual content type than overriding it as application/octet-stream

Reply:
changed as per the review